### PR TITLE
Home

### PR DESCRIPTION
**Pull request by INZDY**
Home page and add new event..`

### CODE REVIEW

**Review by INZDY — COMMENTED** [review]
Some chunks need to be revised.

File: lib/pages/AddNewEvent.dart
Comment (INZDY) [inline lib/pages/AddNewEvent.dart:1]: Move this file to another branch, this is not related to home.

File: lib/main.dart
[the remark this answers is not in the document]
Comment (JPCatze) [reply lib/main.dart:1]: will do this evening & push again

File: lib/pages/home.dart
[the remark this answers is not in the document]
Comment (JPCatze) [reply lib/pages/home.dart:1]: will do this evening & push again

File: lib/pages/AddNewEvent.dart
Comment (JPCatze) [reply lib/pages/AddNewEvent.dart:1]: ok, sir

File: lib/main.dart
[the remark this answers is not in the document]
Comment (JPCatze) [reply lib/main.dart:1]: @INZDY  
Done & pushed

File: lib/pages/home.dart
[the remark this answers is not in the document]
Comment (JPCatze) [reply lib/pages/home.dart:1]: @INZDY  
Done & pushed

**Comment by JPCatze** [comment]
all code need for merging is fixed. I will close this pull request for now.

File: lib/pages/AddNewEvent.dart
Comment (INZDY) [reply lib/pages/AddNewEvent.dart:1]: You haven't move this file to another branch yet

Comment (INZDY) [reply lib/pages/AddNewEvent.dart:1]: You haven't move this file to another branch yet.

**Review by INZDY — COMMENTED** [review]
There is 1 remaining problem
